conftest.py

```diff
@@ -48,18 +48,23 @@ def pytest_addoption(parser):
     pytest = None
 
 
-class PyodideInited:
-    def __call__(self, driver):
-        inited = driver.execute_script(
-            "return window.pyodide && window.pyodide.runPython"
-        )
-        return inited is not None
+def PyodideInited(driver):
+    return driver.execute_script(
+        """
+        return !!(window.PYODIDE_READY || window.PYODIDE_ERROR);
+        """
+    )
+
+
+def PyodideInitGetError(driver):
+    return driver.execute_script(
+        "return window.PYODIDE_ERROR && window.PYODIDE_ERROR.message;"
+    )
 
 
-class PackageLoaded:
-    def __call__(self, driver):
-        inited = driver.execute_script("return window.done")
-        return bool(inited)
+def PackageLoaded(driver):
```

Comment:
Since these are functions they should be words separated by underscores,
```suggestion
def package_loaded(driver):
```
and same for the other two.